lib/models/api/v2/tweet_v2.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:tweet_ui/models/api/v2/entities/media_v2.dart';
+import 'package:tweet_ui/models/api/v2/entities/tweet_entities_v2.dart';
+import 'package:tweet_ui/models/api/v2/user_v2.dart';
+
+part 'tweet_v2.g.dart';
+
+/// Comparing this model to Twitter API V1, it is important to notice that in V2 most of the fields are optional.
+/// The only required fields are id and text. But this package might require some more fields to properly display tweets.
+/// Those need to be included in expansion parameters.
+@JsonSerializable()
+class TweetV2Response {
```

Comment:
Class name does not correspond to file name.
Maybe we can go with `TweerV2` ?

Reply:
There is a class TweetV2 in the file. Naming file this way I wanted to stay consistent with API V1.1